doc/services/shell/index.rst

```diff
@@ -181,7 +181,8 @@ Abstract code for this task would look like this:
         }
 
         SHELL_SUBCMD_DICT_SET_CREATE(sub_gain, gain_cmd_handler,
-                (gain_1, 1), (gain_2, 2), (gain_1_2, 3), (gain_1_4, 4)
+                (gain_1, 1, "gain 1"), (gain_2, 2, "gain 2"),
+		(gain_1_2, 3, "gain 1/2"), (gain_1_4, 4, "gain 1/4")
```

Comment:
This should be indented with tabs as well (the line above already is)

Reply:
So the formating is already indented correctly. If you look at the raw https://raw.githubusercontent.com/ualbertagreen/zephyr/shell_documentation_fix/doc/services/shell/index.rst or even the code snippet quoted in your comment it is displayed as indented correctly. I don't know why in the files changed tab it is displayed incorrectly, but it is a display error on github's side.

Reply:
Done.